src/games/limbo/index.ts

```diff
@@ -140,6 +143,8 @@ export const getLimboResult = async ({
         });
 
         let results: LimboResult[] = [];
+        let rawResults: LimboParsedJson[] = [];
+        let txDigests: string[] = [];
 
```

Comment:
should this be inside an array? i.e. we should only ever get one txDigest per txn and one parsedJson per txn

Reply:
there were only ever be one result, rawResult, and txDigest. i wanted to keep it consistent with the way it had been built previously.

we should change it when we move to native randomness, or update the start game function to accept an array of multiple bets.